### PR TITLE
Array-3 Solutions

### PR DESCRIPTION
**Pull request by thanujkumarj**

### CODE REVIEW

File: problem3.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+// we reverse the given array and then 
+// we reverse firt k elements, then we reverse remaining elements
+
+class Solution {
+public:
+    void rotate(vector<int>& nums, int k) {
+        int n = nums.size();
+        k = k%n;
+        reverse(nums.begin(),nums.end());
```
Comment (ishika2741) [inline problem3.cpp:17]: Where is reverse function?

**Review by ishika2741 — COMMENTED** [review]
Well done! Keep it up.

**Comment by ishika2741** [comment]
Well done! Keep it up.

**Comment by thanujkumarj** [comment]
> Well done! Keep it up.

**Comment by thanujkumarj** [comment]
> Well done! Keep it up.

In cpp, reverse is an stl function. i used it

**Pull request closed without merging**